arangod/Agency/Agent.cpp

```diff
@@ -2513,6 +2513,23 @@ void Agent::updateSomeConfigValues(velocypack::Slice data) {
     _config.setSupervisionGracePeriod(d);
     _supervision->setGracePeriod(d);
   }
+  slice = data.get("delayAddFollower");
+  uint64_t u;
```

Comment:
Same here. Should we use `double` instead?

Reply:
No, the delays are intentional `uint64_t`.